docs/deprecated.md

```diff
@@ -0,0 +1,39 @@
+# Deprecated functionality
+
+Deprecated features are features that iotagent-json stills support but that are
+not maintained or evolved any longer. In particular:
+
+-   Bugs or issues related with deprecated features and not affecting
+    any other feature are not addressed (they are closed in github.com
+    as soon as they are spotted).
+-   Documentation on deprecated features is removed from the repository documentation.
+    Documentation is still available in the documentation set associated to older versions
+    (in the repository release branches).
+-   Deprecated functionality is eventually removed from iotagent-json. Thus you
+    are strongly encouraged to change your implementations using iotagent-json
+    in order not rely on deprecated functionality.
+
+A list of deprecated features and the version in which they were deprecated follows:
+
+* Support to Node.js v4 in iotagent-json 1.9.0. The use of Node.js v8 is highly recommended.
```

Comment:
It would be great to add also "NGSIv1 support" (as https://github.com/telefonicaid/lightweightm2m-iotagent/blob/master/docs/deprecated.md does). Also in the table.